scilpy/gradients/utils.py

```diff
@@ -33,14 +42,16 @@ def random_uniform_on_sphere(nb_vectors):
 
 def get_new_order_philips(philips_table, dwi, bvals, bvecs):
     """
-    Reorder bval and bvec files based on the philips table.
+    Find the sorting order that could be applied to the bval and bvec files to
+    obtain the same order as in the philips table.
```

Comment:
I think this function could be more general. It could be like get_gtab_reorder or something. I would say:

Find the sorting order that brings the bval and bvec files to the same order as in the given gradient table. This is mostly useful to reorder bval and bvec files in the order they were acquired by the Philips scanner (before version 5.6).

Reply:
Thus, I would replace philips_table with gradient_table. (and update the function accordingly by removing all philips)

Reply:
done